SAMtemplates/lambda_resources.yaml

```diff
@@ -409,6 +413,365 @@ Resources:
         EntryPoints:
           - certificateChecker/src/certificateChecker.ts
 
+  ##################################################
+  # Proxygen lambdas
+  ##################################################
+  ExecuteProxygenPTLManagedPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Roles:
+        - !ImportValue ci-resources:CloudFormationDeployRoleName
+      # remove above line and uncomment this once complete
```

Comment:
yes. 
Eventually we want only the proxygen role to have permissions to execute these lambdas
Thhis pr creates ci-resources:ProxygenPTLRoleName in cloudformation/ci_resources. 
Resources in cloudformation formation do not get created in a pull request (as its resources there should only be one of in the AWS account). 
But resources in SAMtemplates DO get created on pull request. 
If I set it to ci-resources:ProxygenPTLRoleName in this pull request, the deployment of SAMtemplates/lambda_resources would fail as  ci-resources:ProxygenPTLRoleName wont exist yet

https://github.com/NHSDigital/electronic-prescription-service-account-resources/pull/307 sets the correct roles and you can see the build has failed. It will work once this is merged